portal/src/components/documentation/icon/IconTable/IconTable.scss

```
@@ -0,0 +1,60 @@
+@import "~@fremtind/jkl-core/variables/_all.scss";
+
+.jkl-icon-table {
+    border: 0;
+    width: 100%;
+    max-width: rem(768px);
+
+    &__data,
+    &__header {
+        padding: $component-spacing--small;
+        @include jkl-text-style("desktop/body");
+        text-align: left;
+        vertical-align: top;
+        border-bottom: rem(1px) solid;
+        box-sizing: border-box;
+    }
+
+    &__header {
+        font-weight: $bold-weight;
+        border-bottom-color: $svart;
+        min-height: rem(48px);
+    }
+
+    &__data {
+        border-bottom-color: $gra-50;
+        padding-bottom: $component-spacing--large;
+        min-height: rem(64px);
+    }
+
+    @include small-device {
+        width: initial;
+
+        &__header {
+            padding-left: rem(125px);
+            &:first-child {
+                padding-left: rem(20px);
+                margin-left: 0;
+            }
+        }
+        &__data {
+            border: none;
+            position: relative;
+            margin-left: rem(25px);
+            padding-left: rem(125px);
+            &:first-child {
+                padding-left: rem(20px);
+                margin-left: 0;
+            }
+            &::before {
+                @include jkl-text-style("desktop/micro");
+                position: absolute;
+                top: 13px;
+                left: 0;
+            }
+        }
+        &__row {
+            border-bottom: 1px solid $gra-50;
+        }
+    }
```

Veldig mange magiske tall her. Foreslår en forenkling av dette:

```suggestion
    @include small-device {
        &__header,
        &__data {
            &:first-child {
                width: 40%; // kan også være en fast verdi om det er bedre 
            }
        }
    }
```